### DISCUSSION TITLE
模型量化成int8之后的尺寸问题

### DISCUSSION THREAD

**Discussion by ZhenyuYangGithub** [Tencent/ncnn#4554] [General]
我的模型量化成int8之后，模型.bin文件的尺寸变成了原来文件的1/2。但理论上讲，fp32的模型量化成int8，应该是4:1的尺寸关系。请问这是什么原因？

**Comment by csukuangfj** [comment]
> 请问这是什么原因？

你要看看你模型里都有些什么 OP.

ncnn 支持对特定的 OP, 如 Linear 和 Conv 等做 int8 量化。

如果你的模型里，只有很少的 OP 支持量化，那么你看不到 4:1 的关系.

**Reply by ZhenyuYangGithub** [reply]
> > 请问这是什么原因？
> 
> 你要看看你模型里都有些什么 OP.
> 
> ncnn 支持对特定的 OP, 如 Linear 和 Conv 等做 int8 量化。
> 
> 如果你的模型里，只有很少的 OP 支持量化，那么你看不到 4:1 的关系.

谢谢，哪里可以查到ncnn支持量化的OP的列表？换句话说，怎样知道模型中的哪些层被量化了？

**Reply by csukuangfj** [reply]
请去看源代码
